docs/source/techniques/CrystalFieldExamples.rst

```diff
@@ -0,0 +1,222 @@
+.. _Crystal_Field_Examples:
+
+======================
+Crystal Field Examples
+======================
+
+Mantid supports the calculation and fitting of inelastic neutron scattering (INS) measurements of transitions between crystal field (or crystalline electric field) energy levels. The scientific background is summarised in the `concept page <https://docs.mantidproject.org/nightly/concepts/index.html>`_ whilst the syntax of the Python commands used is described in the `interface page <https://docs.mantidproject.org/nightly/interfaces/index.html>`_
```

Comment:
Can we use internal Sphinx links to pages that are in this repo? This way we don't hardcode the server address in too many places and the link goes to the correct version rather than the most current nightly.

Can "concept page" & "interface page" go to a specific pages as it's not quite clear here where I would go after that?